src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -194,6 +197,7 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         private profileService: ProfileService,
         private modalService: NgbModal,
         private examParticipationService: ExamParticipationService,
+        private printService: ThemeService,
```

Comment:
I would suggest to keep the naming consistent here

```suggestion
        private themeService: ThemeService,
```

Reply:
removed 👍